tests/e2e/test_api.py

```diff
@@ -1,11 +1,13 @@
 """Integration tests for basic OLS REST API endpoints."""
 
+import uuid
+
 import requests
 from httpx import Client
 
 client = Client(base_url="http://localhost:8080")
 
-conversation_id = "0123456789abcdef0123456789abcdef"
+conversation_id = str(uuid.uuid4())
```

Comment:
it probably doesn't matter here but it's generally preferred to use constants for tests so we get consistent/reproducible results.  Or if you're using randomly generated values, you want to ensure you use a consistent seed for the generator so that the "Random" values are always the same when generated (only when testing of course!)

Reply:
ok not a problem to change, minute

Reply:
updated @bparees